src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -0,0 +1,71 @@
+package edu.kis.vh.nursery.stack;
+
+/**
+ * Class implementing default stack functionality
+ */
+public class IntArrayStack implements Stackable {
+
+	private static final int STACK_CAPACITY = 12;
+
+	private final int[] numbers = new int[STACK_CAPACITY];
+
+	/**
+	 * Gets total amount of numbers placed on stack minus 1
+	 * @return total amount of numbers placed on stack minus 1
+	 */
+	@Override
+	public int getTotal() { return total; }
+
+	private int total = EMPTY_STACK_VALUE-1;
```

Comment:
Yes, total variable initialization and validation should always remain -1, but:
Changing EMPTY_STACK_VALUE value requires such fixes? It does not make sense. Incorrect fix.